opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp

```diff
@@ -1254,6 +1275,17 @@ namespace {
         struct Chop01 {
             double operator()(double x) const { return std::max(std::min(x, 1.0), 0.0); }
         };
+
+        double infinityNorm( const ADB& a )
+        {
+            if( a.value().size() > 0 ) {
+                return a.value().matrix().template lpNorm<Eigen::Infinity> ();
```

Comment:
Actually it was correct in the initial version since there it was in a template context. I did not notice this either, I guess that's what we have compilers for... I am surprised that 4.8 does not complain, my clang accepts it also. Does that indicate that this has been relaxed a little in C++11 or C++14? It does not matter much since we must compile on gcc 4.4 anyway.